sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/Samples/BindingToCollector.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs;
+
+namespace Microsoft.Azure.WebJobs.Extensions.EventHubs.Tests.Samples
+{
+    public static class BindingToCollector
+    {
+        #region Snippet:BindingToCollector
+        [FunctionName("BindingToCollector")]
+        public static async Task Run(
+            [TimerTrigger("0 */5 * * * *")] TimerInfo myTimer,
+            [EventHub("<event_hub_name>", Connection = "<connection_name>")] IAsyncCollector<EventData> collector)
+        {
+            // IAsyncCollector allows sending multiple events in a single function invocation
+            await collector.AddAsync(new EventData(new BinaryData($"Event 1 added at: {DateTime.Now}")));
```

Comment:
Just a note:  `EventData` structure changed to include `BinaryData` between the last GA (5.2.0) and the current beta family (5.3.0).    If you're looking to represent the current GA state, you'll want to use the hidden `ReadOnlyMemory<byte>` constructor overload.

Reply:
I'll keep it targeting 5.3 for now but will confirm that dates align.